doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -5,17 +5,88 @@
 
 ## Table of Contents
 * [3.1 Introduction](#3.1)
-* [3.2 Heading](#3.2)
-* [3.3 Heading](#3.3)
+* [3.2 Measurement](#3.2)
+  * [3.2.1 Performance & Resiliency Measurements](#3.2.1)
+  * [3.2.2 Storage and IOPS](#3.2.2)
+* [3.3 NFVI Test Cases](#3.3)
 
 <a name="3.1"></a>
 ## 3.1 Introduction
 
 > Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
 
 <a name="3.2"></a>
-## 3.2 Heading
+## 3.2 Measurement
 
+Test validations will be corroborated, and confirmed, with direct comparison between measured results and documented non-functional requirements (NFRs) for applications, hardware and software configuration settings, and host systems.  Throughput, latency, concurrent connections/threads, are all examples of non-functional requirements which specify criteria which can be used to judge the operation of a system, rather than specific behaviours of the application which are defined by functional requirements.
+
+This section attempts to summarize a categorical list of metrics used for test validations.  **For a complete list of metrics, and requirements, please refer to Reference Model**
+
+<a name="3.2.1"></a>
+### 3.2.1 Performance & Resiliency Measurements
```

Comment:
I suggest to separate those chapters as requirements and criteria differ.

Reply:
do you have a suggestion @gkunz where to place them?

Reply:
nothing fancy: keep it in chapter03 but dedicate a separate heading to the resilience measurements

Reply:
resolved in latest commits